mne/preprocessing/maxwell.py

```diff
@@ -502,6 +579,7 @@ def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.,
                          for coil in coils])] = mag_scale
 
     # Compute internal/external basis vectors (exclude degree 0; L/RHS Eq. 5)
+    mu_0 = 4e-7 * np.pi  # magnetic permeability
```

Comment:
Might add a note here that Elekta may not be using the actual irrational value. Samu's code used a truncated number. Near zero percent chance it matters, but a comment is cheap :p